fmdtools/analyze/graph.py

```diff
@@ -1147,41 +1360,99 @@ def __init__(self, mdl, withstates=True, **kwargs):
         **kwargs : kwargs
             (placeholder for kwargs)
         """
-        self.g=self.nx_from_obj(mdl)
-        if withstates: self.set_nx_states(mdl)
+        self.g = self.nx_from_obj(mdl)
+        if withstates:
+            self.set_nx_states(mdl)
+
     def nx_from_obj(self, mdl):
-        g=mdl.graph.copy()
-        labels = {fname:f.get_typename() for fname, f in mdl.fxns.items()}
-        labels.update({fname:f.get_typename() for fname, f in mdl.flows.items()})
+        """
+        TODO Needs documentation
+        Parameters
+        ----------
+        mdl
```

Comment:
Generates the networkx.graph object corresponding to the model.

mdl: Model
     Model to create the graph representation of.

Returns
--------
g : networkx.Graph
     networkx.Graph representation of model functions and flows (along with their attributes)

Reply:
Done